### PR TITLE
brew audit fixes

### PR DESCRIPTION
**Pull request by i-freelancer**
### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>`?

### Changes to Homebrew's Core:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031) if you'd like one.
- [ ] Have you successfully ran `brew tests` with your changes locally?

### CODE REVIEW

**Comment by nijikon** [comment]
Commit message should be ```<formula>: audit fixes```

**Comment by nijikon** [comment]
Also, one commit per formula

File: Library/Formula/sleepwatcher.rb [hunk 1]
```diff
@@ -18,8 +18,8 @@ def install
     end
 
     # Build and install binary
-    cd "sources" do
-      system "mv", "../sleepwatcher.8", "."
+    Dir.chdir("sources") do
```
Comment (bfontaine) [inline Library/Formula/sleepwatcher.rb]: Please keep `cd` here.

File: Library/Formula/geocouch.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@ def install
     #  Install geocouch build files.
     (share/"geocouch").mkpath
     rm_rf share/"geocouch/ebin/"
-    (share/"geocouch").install Dir["ebin"]
+    (share/"geocouch").install ebin
```
Comment (bfontaine) [inline Library/Formula/geocouch.rb]: This won’t work because the `ebin` variable doesn’t exist.

File: Library/Formula/geocouch.rb [hunk 3]
```diff
@@ -59,7 +59,7 @@ def install
     (share/"geocouch/geocouch.plist").chmod 0644
 
     #  Install geocouch.ini into couchdb.
-    (etc/"couchdb/default.d").install Dir["etc/couchdb/default.d/geocouch.ini"]
+    (etc/"couchdb/default.d").install etc/couchdb/default.d/geocouch.ini
```
Comment (bfontaine) [inline Library/Formula/geocouch.rb]: This needs quotes to be a string (`"etc/couchdb/default.d/geocouch.ini"`).

File: Library/Formula/geocouch.rb [hunk 4]
```diff
@@ -78,7 +78,7 @@ def install
     (couchdb_share/"www/script/test/geocouch").mkpath
     (couchdb_share/"www/script/test/geocouch").install test_files
     Dir[(couchdb_share/"www/script/test/geocouch/*.js")].each  \
-      { |geotest| system "cd #{couchdb_share/"www/script/test"};  ln -s geocouch/#{File.basename(geotest)} ." }
+      { |geotest| Dir.chdir("#{couchdb_share/"www/script/test"}"); ln("ln -s geocouch/#{File.basename(geotest)} .") }
```
Comment (bfontaine) [inline Library/Formula/geocouch.rb]: This changes the original behavior which was to make a symlink without changing the current process’ directory.

File: Library/Formula/sleepwatcher.rb [hunk 5]
```diff
@@ -18,8 +18,8 @@ def install
...
+      move("../sleepwatcher.8", ".")
```
Comment (bfontaine) [inline Library/Formula/sleepwatcher.rb]: Please use `mv` and drop the parentheses.

**Comment by i-freelancer** [comment]
Thank you.I will work on this and ensure that these errors do not occur in future.

**Pull request closed without merging**